payments-core/src/main/java/com/stripe/android/cards/RemoteCardAccountRangeSource.kt

```diff
@@ -26,23 +25,20 @@ internal class RemoteCardAccountRangeSource(
         cardNumber: CardNumber.Unvalidated
     ): List<AccountRange>? {
         return cardNumber.bin?.let { bin ->
-            _loading.value = true
+            val accountRanges = withLoading {
+                stripeRepository.getCardMetadata(bin, requestOptions)?.accountRanges
+            }
 
-            val accountRanges =
-                stripeRepository.getCardMetadata(bin, requestOptions)?.accountRanges.orEmpty()
-            cardAccountRangeStore.save(bin, accountRanges)
```

Comment:
We no longer store _any_ response, as these could include `null` responses due to network errors. Instead, the repository will decide whether to store a result.